Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Follow.cs

```diff
@@ -0,0 +1,644 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Solvers
+{
+    /// <summary>
+    /// Follow solver positions an element relative in front of the forward axis of the reference.
+    /// The element can be loosly constrained (a.k.a. tag-along) so that it doesn't follow until it is too far.
+    /// </summary>
+    public class Follow : Solver
```

Comment:
you could add it to the components menu as well   [AddComponentMenu("Scripts/MRTK/Experimental/Solver/Follow")]